token/cli/src/output.rs

```diff
@@ -683,21 +699,47 @@ fn display_ui_extension(
         }
         // ExtensionType::Uninitialized is a hack to ensure a mint/account is never the same length as a multisig
         UiExtension::Uninitialized => Ok(()),
-        UiExtension::ConfidentialTransferMint(_) => writeln_name_value(
-            f,
-            "    Unparseable extension:",
-            "ConfidentialTransferMint is not presently supported",
-        ),
         UiExtension::ConfidentialTransferAccount(_) => writeln_name_value(
             f,
-            "    Unparseable extension:",
+            "  Confidential transfer:",
             "ConfidentialTransferAccount is not presently supported",
         ),
-        _ => writeln_name_value(
-            f,
-            "    Unparseable extension:",
-            "Consider upgrading to a newer version of spl-token",
-        ),
+        _ => {
+            if let Some(ui_confidential_transfer_extension) = ui_confidential_transfer_extension {
+                match ui_confidential_transfer_extension {
+                    UiConfidentialTransferExtension::ConfidentialTransferMint(ui_mint) => {
+                        writeln!(f, "  {}", style("Confidential transfer:").bold())?;
+                        writeln!(
+                            f,
+                            "    {}: {}",
+                            style("Account approve policy").bold(),
+                            if ui_mint.auto_approve_new_accounts {
+                                "auto"
+                            } else {
+                                "manual"
+                            },
+                        )?;
+                        writeln!(
+                            f,
+                            "    {}: {}",
+                            style("Audit key:").bold(),
+                            if let Some(auditor_pubkey) = ui_mint.auditor_encryption_pubkey.as_ref()
+                            {
+                                auditor_pubkey
+                            } else {
+                                "audits are not enabled"
```

Comment:
nit: should we just say "disabled" here?

Reply:
Yep, good idea!